IPython/utils/PyColorize.py

```diff
@@ -44,12 +44,7 @@
 import token
 import tokenize
 
-try:
-    generate_tokens = tokenize.generate_tokens
-except AttributeError:
-    # Python 3. Note that we use the undocumented _tokenize because it expects
-    # strings, not bytes. See also Python issue #9969.
-    generate_tokens = tokenize._tokenize
+generate_tokens = tokenize.tokenize
```

Comment:
This was `_tokenize`, not `tokenize` for str/bytes distinctions. If we want to use `.tokenize`, we need to pass bytes, or wrap it in an encoder.